docs/user-guide/configuring-containers.md

```diff
@@ -0,0 +1,131 @@
+
+## Environment variables and variable expansion
+
+Kubernetes [does not automatically run commands in a shell](https://github.com/GoogleCloudPlatform/kubernetes/wiki/User-FAQ#use-of-environment-variables-on-the-command-line) (not all images contain shells). If you would like to run your command in a shell, such as to expand environment variables (specified using `env`), you could do the following:
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: hello-world
+spec:  # specification of the pod’s contents
+  restartPolicy: Never
+  containers:
+  - name: hello
+    image: "ubuntu:14.04"
+    env:
+    - name: MESSAGE
+      value: "hello world"
+    command: ["/bin/sh","-c"]
+    args: ["/bin/echo \"${MESSAGE}\""]
+```
+
+However, a shell isn’t necessary just to expand environment variables. Kubernetes will do it for you if you use [`$(ENVVAR)` syntax](../docs/design/expansion.md):
+```yaml
+    command: ["/bin/echo"]
+    args: ["$(MESSAGE)"]
+```
+## Viewing pod status
+
+You can see the pod you created (actually all of your cluster's pods) using the `get` command. 
+
+If you’re quick, it will look as follows:
+```bash
+$ kubectl get pods
+NAME          READY     STATUS    RESTARTS   AGE
+hello-world   0/1       Pending   0          0s
+```
+Initially, a newly created pod is unscheduled -- no node has been selected to run it. Scheduling happens after creation, but is fast, so you normally shouldn’t see pods in an unscheduled state unless there’s a problem.
+
+After the pod has been scheduled, the image may need to be pulled to the node on which it was scheduled, if it hadn’t be pulled already. After a few seconds, you should see the container running:
+```bash
+$ kubectl get pods
+NAME          READY     STATUS    RESTARTS   AGE
+hello-world   1/1       Running   0          5s
+```
+The `READY` column shows how many containers in the pod are running.
+
+Almost immediately after it starts running, this command will terminate. `kubectl` shows that the container is no longer running and displays the exit status:
+```bash
+$ kubectl get pods
+NAME          READY     STATUS       RESTARTS   AGE
+hello-world   0/1       ExitCode:0   0          15s
+```
+## Viewing pod output
+
+You probably want to see the output of the command you ran. As with [`docker logs`](https://docs.docker.com/userguide/usingdocker/), `kubectl logs` will show you the output:
+```bash
+$ kubectl logs hello-world
+hello world
+```
+## Deleting pods
+When you’re done looking at the output, you should delete the pod:
+```bash
+$ kubectl delete pod hello-world
+pods/hello-world
+```
+As with `create`, `kubectl` prints the resource type and name of the resource deleted when successful.
+
+You can also use the resource/name format to specify the pod:
+```bash
+$ kubectl delete pods/hello-world
+pods/hello-world
+```
+
+Terminated pods aren’t currently automatically deleted, so that you can observe their final status, so be sure to clean up your dead pods. 
```

Comment:
*Until you do,you won't be able to create a new pod with the same name, and you will see a more clutter when you do `kubectl get pods`. *